### DISCUSSION TITLE
Trouble running dockerhub_run.sh

### DISCUSSION THREAD

**Discussion by michaellwood** [alshedivat/al-folio#796] [Q&A]
whenever I try to run the following:

![image](https://user-images.githubusercontent.com/41761599/182178977-d1a702b7-2ffa-496d-8400-437c2aeae8e6.png)

I get this error: 

"./bin/dockerhub_run.sh: line 8: syntax error: unexpected end of file"

Any ideas?

**Comment by pourmand1376** [comment]
That's odd. 

Can you run this command?
```bash
docker run --rm -v "$PWD:/srv/jekyll/" -p "8080:8080" \
                    -it amirpourmand/al-folio bundler  \
                    exec jekyll serve --watch --port=8080 --host=0.0.0.0 
```

**Reply by pourmand1376** [reply]
Actually I searched for this, it seems there is some problem with end of file. Can you describe what version are you using?
Does your problem solve when changing end of line format from CRLF to LF?

https://stackoverflow.com/questions/9886268/shell-script-syntax-error-unexpected-end-of-file
